builder/dockerfile/internals.go

```diff
@@ -137,6 +171,53 @@ func (b *Builder) performCopy(state *dispatchState, inst copyInstruction) error
 	return b.exportImage(state, imageMount, runConfigWithCommentCmd)
 }
 
+func parseChownFlag(flChown string) (string, string) {
+	parts := strings.Split(flChown, ":")
+	if len(parts) == 1 {
+		// if no group specified, use the user spec as group as well
+		return parts[0], parts[0]
+	}
+	return parts[0], parts[1]
+}
+
+func lookupUser(userStr, filepath string) (int, error) {
```

Comment:
I think this already exists as `idtools.LookupUser()` / `LookupUID()`? Can we re-use either of those?